tests/unit/api/bucketPutACL.js

```diff
@@ -24,6 +24,18 @@ const canonicalIDforSample1 =
 const canonicalIDforSample2 =
     '79a59df900b949e55d96a1e698fbacedfd6e09d98eacf8f8d5218e7cd47ef2bf';
 
+const invalidIds = {
```

Comment:
It seems to be the standard to place these sorts of test arrays/objects at the top of the file which by all means be consistent, but I would say when reading the test it is very convenient having them together. 200 lines is more than most monitors so it would require scrolling to see or update the test values. 

My usual method is to just use the object directly like...
```javascript
for (const [testName, testValue] of Object.entries({
  'sample test': 'blah',
  'another test': 'blah 2',
}) {
    it(`should do a thing ${testName}`, () => assert(testValue))
}
```

Just an idea to keep tests as close together as possible so scrolling around isn't required to look at the tests.